nevergrad/optimization/es.py

```diff
@@ -64,13 +73,21 @@ def _internal_tell_candidate(self, candidate: p.Parameter, value: float) -> None
             else:
                 self._waiting.append(candidate)
             if len(self._waiting) >= self._config.offsprings:
-                choices = self._waiting + ([] if self._config.only_offsprings else list(self._population.values()))
-                choices.sort(key=lambda x: x._meta["value"])
-                self._population = {x.uid: x for x in choices[:self._config.popsize]}
-                self._uid_queue.clear()
-                self._waiting.clear()
-                for uid in self._population:
-                    self._uid_queue.tell(uid)
+                self._simple_selection()
+
+
+    def _simple_selection(self):
```

Comment:
```suggestion
                self._select()


    def _select(self):
```

I put a lot of care to naming stuff precisely enough because it saves a lot of maintenance time.